### PR TITLE
Cell line element

### PR DESCRIPTION
**Pull request by FabianMauz**
Add user documentation for the new core element : cell line

### CODE REVIEW

File: docs/eln/ui/elements/cell_lines.mdx [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+title: Cell lines
+sidebar_position: 4
+---
+
+## Meaning
+A cell line is a general term defining specific population of cells that can maintain in vitro culture for an extended period of time, and is usually clonal meaning that the entire population originated from a single, common, ancestor cell. <sup>1</sup>
+
+## Model
+A cell line object in chemotion eln is divided into a conceptual and a physical part. The conceptual part describes general information like the name of the cell line, the origin or the source organism. The physical part describes a concrete realization of the conceptual part and can be seen as a real object with properties like number, location or current passage. Multiple physical objects can share the same conceptual part.
```
Comment (endevor-music) [inline docs/eln/ui/elements/cell_lines.mdx:10]: chemotion eln -> Chemotion ELN

File: docs/eln/ui/elements/cell_lines.mdx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+property name |description                         |optional|
+|-----|--|---|
+|amount|numerical or scientific notation|yes      |
+|unit|predefined or custom unit of amount|yes      |
+|contamination        |any known contamination of cell line sample|yes      |
+|name of specific probe|can be used to identify specific probes|yes      |
+|description|any aditional information|yes|
+
+The represantation of the amount can be switched with <img
```
Comment (endevor-music) [inline docs/eln/ui/elements/cell_lines.mdx:76]: represantation -> representation

File: docs/eln/ui/elements/cell_lines.mdx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+2. The button <img
+  src={require("@site/static/img/user_interface/elements/cell_lines/20231204_cell_line_create_button.png").default}
+  alt={"Item specific properties"}
+/> can be used to create a new sample for an existing cell line. To do this, the collection of the cell line sample must be active.
+
+3. By importing a collection with cell lines, cell line samples can be created.
+
+## Import/Export
+
+Collections with cell lines can be exported and imported as with other elements
```
Comment (endevor-music) [inline docs/eln/ui/elements/cell_lines.mdx:108]: add period at end of sentence

File: docs/eln/ui/elements/cell_lines.mdx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+## Search
+
+Cell lines can be searched for and found using the basic search. The following properties are taken into account in the search: name of the cell line and the specific name of the sample
```
Comment (endevor-music) [inline docs/eln/ui/elements/cell_lines.mdx:111]: add period at end of sentence

Comment (FabianMauz) [reply docs/eln/ui/elements/cell_lines.mdx]: fixed

**Review by harivyasi — APPROVED** [review]

**Pull request merged**